backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -301,7 +305,7 @@
             Id = semanticDomain.Guid,
             Name = FromLcmMultiString(semanticDomain.Name),
             Code = semanticDomain.Abbreviation.UiString ?? "",
-            Predefined = true, // TODO: Look up in a GUID list of predefined data
+            Predefined = CanonicalGuidsSemanticDomain.CanonicalSemDomGuids.Contains(semanticDomain.Guid),
```

Comment:
there should be similar code for PartOfSpeech, we should do the same thing there

Reply:
Done.